server/server.go

```diff
@@ -177,6 +178,14 @@ func (s *server) Run(rootCtx context.Context) (err error) {
 
 		s.gRPCServer.Stop()
 		zap.L().Warn("closed gRPC server")
+
+		if s.network != nil {
+			stopCtx, stopCtxCancel := context.WithTimeout(context.Background(), StopOnSignalTimeout)
+			defer stopCtxCancel()
+			s.network.stop(stopCtx)
+			zap.L().Warn("stoping network")
```

Comment:
Is `stop` async operation? Should we add warning before we call `stop`?

Reply:
I think it's not async from here.

Reply:
stop is blocking. changed msg to "network stopped"